plutus-pab/put-data-in-pab.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/env bash
```

Comment:
not sure what to do about the two scripts - it would be nice to have them in the repo, but they might get out of date really quickly if we just put them like this. The best approach IMO would be to add them as nix scripts and then run them as part of the VM tests. This could be a separate PR though.

Reply:
sounds good @j-mueller :)

i'd also be happy to temporarily put them in a `hacky-scripts` folder to imply that they aren't expected to be long-lived/supported.

Reply:
Ok, since you're going to do the VM tests right after this one we can leave the scripts in for now.

Reply:
https://jira.iohk.io/browse/SCP-2385